diff --git a/python_wrapper/CMakeLists.txt b/python_wrapper/CMakeLists.txt
index f1686d0..ba0f97b 100644
--- a/python_wrapper/CMakeLists.txt
+++ b/python_wrapper/CMakeLists.txt
@@ -25 +25 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../lib/python/${A
-add_subdirectory(third-party/pybind11)
+find_package(pybind11 CONFIG REQUIRED)
@@ -28,2 +28 @@ pybind11_add_module(robot_interface python_interface.cpp)
-target_link_libraries(robot_interface ${EXTRA_LIBS} unitree_legged_sdk)
-
+target_link_libraries(robot_interface PRIVATE ${EXTRA_LIBS} unitree_legged_sdk)
